tests/test_hist.py

```diff
@@ -0,0 +1,41 @@
+import pytest
+import numpy as np
+from altairexpress import altairexpress as ae
+from gapminder import gapminder
+
+# define testing data
+gapminder
+
+
+def test_hist(gapminder):
```

Comment:
Don't think test function should take inputs, just load the gap minder inside the function